lib/whitehall_importer/create_edition.rb

```diff
@@ -194,5 +196,66 @@ def create_revision_history(edition)
         )
       end
     end
+
+    def create_notes(edition)
+      whitehall_edition["editorial_remarks"].each do |event|
+        details = TimelineEntry::WhitehallImportedEntry.create!(
+          entry_type: "internal_note",
+          body: {
+            "body": event["body"],
+          }.to_json,
+        )
+        TimelineEntry.create!(
+          entry_type: :whitehall_migration,
+          created_at: event["created_at"],
+          created_by_id: user_ids[event["author_id"]],
+          edition: edition,
+          document: edition.document,
+          details: details,
+        )
+      end
+    end
+
+    def create_fact_checks(edition)
+      whitehall_edition["fact_check_requests"].each do |event|
+        create_fact_check_request(edition, event)
+        create_fact_check_response(edition, event) if event["comments"].present?
+      end
+    end
+
+    def create_fact_check_request(edition, event)
+      details = TimelineEntry::WhitehallImportedEntry.create!(
+        entry_type: "fact_check_request",
+        body: {
+          "email_address": event["email_address"],
+          "instructions":  event["instructions"],
```

Comment:
We've got some excess whitespace here.

Reply:
Sorry, lining up is a personal pref, will adjust my brain ;-)